sympy/stats/rv.py

```diff
@@ -1385,3 +1395,49 @@ def _value_check(condition, message):
     if truth == False:
         raise ValueError(message)
     return truth == True
+
+def _symbol_converter(sym):
```

Comment:
This looks more like code for sympy/core... anyway, you sure you need this?

Reply:
Yes, it's used a lot of times. Though I will check if there's a pre-existing function.

Reply:
I don't think there is any pre-existing function. I asked in the gitter room `sympy/sympy`.
Let this utility function be here since, it is used only by this module. If needed we will shift to `sympy/core` later sometime.